doc/services/portability/posix/option_groups/index.rst

```diff
@@ -367,14 +367,14 @@ POSIX_SEMAPHORES
    :header: API, Supported
    :widths: 50,10
 
-    sem_close(),
+    sem_close(),yes
```

Comment:
Just one minor nit - the doc changes should probably be in a separate commit.

Reply:
fixed